### DISCUSSION TITLE
支持的 Node.js 版本

### DISCUSSION THREAD

**Discussion by FrzMtrsprt** [1715173329/UnblockNeteaseMusic#978] [Q&A] [answered]
请问UNM Enhanced支持使用哪些Node.js版本运行呢？最近在Node.js 18和19上运行脚本时，遇到了网易云测试代理不通，各种数据也无法加载的问题，但使用Node.js 16打包的可执行文件就可以测试代理成功，各种数据也可以成功加载，请问是UNM Enhanced不支持Node.js 18和19吗？我不太清楚是不是我本地环境的问题。

**Comment by 1715173329** [comment, marked as answer]
Node.js 18 测试过是正常的。
